pkg/gofr/gofr.go

```diff
@@ -250,8 +250,23 @@ func (a *App) Logger() logging.Logger {
 
 // SubCommand adds a sub-command to the CLI application.
 // Can be used to create commands like "kubectl get" or "kubectl get ingress".
-func (a *App) SubCommand(pattern string, handler Handler) {
-	a.cmd.addRoute(pattern, handler)
+func (a *App) SubCommand(pattern string, handler Handler, help string, description ...string) {
```

Comment:
@IvanGael this is still a breaking change, we cannot have `help string` as parameter. Please stick to the variadic input `description`. Keep it simple, if descriptions are provided, just join them and make it as help doc for that sub command.

```
func (a *App) SubCommand(pattern string, handler Handler, description ...string) {
    if len(description) > 0 {
        desc := description[0]
    }

    help := strings.Join(description, "\n")
}
```

Something like this should suffice

Reply:
@IvanGael please note the code provided here is invalid

It was provided as is, but you will have to rework it `desc` must be defined before the `if description`. Otherwise, there is a scope issue


> ```go
> func (a *App) SubCommand(pattern string, handler Handler, description ...string) {
>     if len(description) > 0 {
>         desc := description[0]
>     }
> 
>     help := strings.Join(description, "\n")
> 
>     a.cmd.addRoute(route{
> 		pattern:     pattern,
> 		handler:     handler,
> 		description: desc,
>                  help:   help,
>     }
> }
> ```